MinPlatformPkg/Include/Library/PlatformConfigCheckLib.h

```diff
@@ -0,0 +1,26 @@
+/** @file
+
+  Platform configuration check library for TestPointCheckLib
+
+  Copyright (c) Microsoft Corporation. All rights reserved
```

Comment:
These files should have the normal BSD-2-Clause-Patent license if going into MinPlatformPkg.